### DISCUSSION TITLE
Where is ToBitmap() in version 7.20.0.1

### DISCUSSION THREAD

**Discussion by Gerald-W** [ImageMagick/ImageMagick#2236] [Help] [answered]
Hi,
I have this code in my program:
using MagickImage image = new MagickImage(array, MagickFormat.Jpeg);
. . . .
. . . 
pictureBox1.Image = image.ToBitmap();

This worked with Magick.NET-Q16-AnyCPU version 7.16.0, but after updating to 7.20.0.1 I get the following error.
'MagickImage' does not contain a definition for 'ToBitmap' and no accessible extension method 'ToBitmap' accepting a first argument of type 'MagickImage' could be found (are you missing a using directive or an assembly reference?'

Where can I find the new syntax for converting a byte array to a jpg? 
Thank you.

**Comment by dlemstra** [comment, marked as answer]
You are asking the question in the wrong project. The answer to your question can be found here: https://github.com/dlemstra/Magick.NET/discussions/652.
